CONFIGDOC.md

```diff
@@ -273,7 +273,7 @@ Input                     | Description
 `sourceFolder`            | (Optional) Location of the folder containing source code, relative to the project root folder. The specified folder and its subfolders are added to the top of the MATLAB search path. If you specify `sourceFolder` and then generate a coverage report, the plugin uses only the source code in the specified folder and its subfolders to generate the report.<br/>**Example:** `['source']`<br/>**Example:** `['source/folderA', 'source/folderB']`
 `selectByFolder`          | (Optional) Location of the folder used to select test suite elements, relative to the project root folder. To create a test suite, the plugin uses only the tests in the specified folder and its subfolders.<br/>**Example:** `['test']`<br/>**Example:** `['test/folderA', 'test/folderB']`
 `selectByTag`             | (Optional) Test tag used to select test suite elements. To create a test suite, the plugin uses only the test elements with the specified tag.<br/>**Example:** `'FeatureA'`
-`testResultsPDF`          | (Optional) Path to write test results report in PDF format. Currently, this input is not supported on macOS platforms.<br/>**Example:** `'test-results/results.pdf'`      
+`testResultsPDF`          | (Optional) Path to write test results report in PDF format.<br/>**Example:** `'test-results/results.pdf'`      

```

Comment:
Can't we say this?

On macOS platforms, this input is supported in MATLAB R2020b and later.

Reply:
Done.